HPXMLtoOpenStudio/measure.rb

```diff
@@ -1895,7 +1916,13 @@ def self.add_unmet_hours_output(model, spaces)
       end
       line += " && (#{@hvac_availability_sensor.name} == 1)" if not @hvac_availability_sensor.nil?
       program.addLine(line)
-      program.addLine("  Set #{htg_hrs} = #{htg_hrs} + #{htg_sensor.name}")
+      if is_ddb_control
+        program.addLine("  If #{zone_air_temp_sensor.name} < (#{sensor_htg_spt.name} - #{UnitConversions.convert(hvac_control.onoff_thermostat_deadband, 'deltaF', 'deltaC')})")
```

Comment:
Modify the unmet hour EMS to exclude time steps when temperature is within deadband.